Packs/Core/Playbooks/playbook-Netcat_Makes_or_Get_Connections.yml

```diff
@@ -0,0 +1,955 @@
+    type: title
+    task:
+      id: bfbce647-7068-4dbc-8450-d3b7436aeca4
+      version: -1
+      name: Investigation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "39"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1450
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "25":
+    id: "25"
+    taskid: 939d4df1-3b9d-4a02-8634-59f622bffba6
+    type: regular
+    task:
+      id: 939d4df1-3b9d-4a02-8634-59f622bffba6
+      version: -1
+      name: Check Previous similar Alerts
+      description: |-
+        Finds past similar alerts based on alert fields' similarity. Includes an option to also display indicators similarity.
+        Note: For the similarity calculation, at least one field must be provided in one of the "similarTextField", "similarCategoricalField", or "similarJsonField" arguments.
+      scriptName: DBotFindSimilarAlerts
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "33"
+    scriptarguments:
+      query:
+        simple: resolution_status:"STATUS_060_RESOLVED_FALSE_POSITIVE"
+      similarTextField:
+        simple: name
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -275
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "27":
+    id: "27"
+    taskid: 904dd046-4637-4204-87b8-2f5e4a4eb7dd
+    type: regular
+    task:
+      id: 904dd046-4637-4204-87b8-2f5e4a4eb7dd
+      version: -1
+      name: Get Extra Data for DNS query name
+      description: Returns information about each alert ID.
+      script: '|||core-get-cloud-original-alerts'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "29"
+    scriptarguments:
+      alert_ids:
+        complex:
+          root: alert
+          accessor: id
+          transformers:
+          - operator: uniq
+      filter_alert_fields:
+        simple: "false"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 75
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: 05b1c3b4-24f9-4033-89dc-fa548b12a39d
+    type: condition
+    task:
+      id: 05b1c3b4-24f9-4033-89dc-fa548b12a39d
+      version: -1
+      name: Destination IP is External?
```

Comment:
Do we want to automatically mark this as FP if it's internal address?

Reply:
We already passed the similar FP check, at this point it might be illegitimate usage.